data/darktableconfig.xml.in

```diff
@@ -67,6 +67,13 @@
     <shortdescription>enable shortcut fallbacks</shortdescription>
     <longdescription>enables default meanings for additional buttons, modifiers or moves when used in combination with a base shortcut</longdescription>
   </dtconfig>
+  <dtconfig prefs="misc" section="interface">
+    <name>accel/load_defaults</name>
+    <type>bool</type>
+    <default>true</default>
+    <shortdescription>restore default shortcuts at startup</shortdescription>
```

Comment:
Perhaps "load" instead of "restore", since the latter kind of suggests user settings will be overwritten